README.MD

```diff
@@ -0,0 +1,29 @@
+# Team 937's code for the 2024 FRC season
+
+[![The Compile State of this Code](https://github.com/frc937/robot2024/actions/workflows/compile.yml/badge.svg)](https://github.com/frc937/robot2024/actions/workflows/compile.yml)
+
+### The Spec Sheet for our Robot
+![A picture of team 937's spec sheet for the robot.](./pictures/Robot%20Specs.png)
```

Comment:
you will prooooobably need to change this url? idk how github markdown works @QuackitsQuinn help me